javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/javaparsermodel/contexts/CompilationUnitContext.java

```diff
@@ -179,7 +199,7 @@ public CompilationUnitContext(CompilationUnit wrappedNode, TypeSolver typeSolver
                 if (importDecl.isAsterisk()) {
                     String qName = importDecl.getNameAsString() + "." + name;
                     SymbolReference<ResolvedReferenceTypeDeclaration> ref = typeSolver.tryToSolveType(qName);
-                    if (ref.isSolved()) {
+                    if (ref != null && ref.isSolved()) {
```

Comment:
why should that be null? I think it should not

Reply:
See below.